lib/dynamodb-document-client.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const assert = require('assert');
+
+const AwsDriverError = require('./aws-driver-error');
+const Driver = require('./driver');
+
+class DynamoDBDocumentClientDriver extends Driver {
+  constructor(options) {
+    const clientOpts = {
+      apiVersion: '2012-08-10'
+    };
+    super('DynamoDB.DocumentClient', clientOpts, options);
+  }
+
+  async batchGet(context, params) {
+    const results = {};
+    let callParams = Object.assign({}, params);
+    let attempts = 0;
+
+    /* eslint-disable no-await-in-loop */
+    while (attempts < (this._conf.attempts || 10)) {
```

Comment:
should we pick a 'safer' default value, because 10 could mean 10 *10 attempts as the aws-sdk retries too

Reply:
I don't mind, but there is always a chance that you'll need a lot of retries to fetch all items, depending on their size. Not sure if this should be guarded by the number of attempts at all.